ota-plus-web/app/reactapp/src/components/campaigns/ListHeader.jsx

```diff
@@ -4,30 +4,29 @@ import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import { observer } from 'mobx-react';
 
+import { CAMPAIGNS_STATUS_PREPARED } from '../../config';
+
 @observer
 class ListHeader extends Component {
   static propTypes = {
-    status: PropTypes.string,
+    status: PropTypes.string.isRequired,
   };
 
   render() {
     const { status } = this.props;
-    const headline = {
-      prepared: 'In preparation',
-      launched: 'running',
-      finished: 'finished',
-      cancelled: 'canceled',
-    };
-    const showColumns = status !== 'prepared';
+    const showColumns = status !== CAMPAIGNS_STATUS_PREPARED;
 
     return (
-      <div className='campaigns__header'>
-        <div className='campaigns__column'>{headline[status]}</div>
-        {showColumns && <div className='campaigns__column'>{'Created at'}</div>}
-        {showColumns && <div className='campaigns__column'>{'Processed'}</div>}
-        {showColumns && <div className='campaigns__column'>{'Affected'}</div>}
-        {showColumns && <div className='campaigns__column'>{'Finished'}</div>}
-        {showColumns && <div className='campaigns__column'>{'Failure rate'}</div>}
+      <div className="campaigns__header">
+        <div className="campaigns__column">{'Name'}</div>
+        {showColumns && <div className="campaigns__column">{'Created at'}</div>}
```

Comment:
I guess this could be a little cleaner

```suggestion
        {showColumns && (
          <>
            <div className="campaigns__column">{'Created at'}</div>
            <div className="campaigns__column">{'Status'}</div>
            <div className="campaigns__column">{'Selected devices'}</div>
            <div className="campaigns__column">{'Failed %'}</div>
            <div className="campaigns__column">{'Successful %'}</div>
            <div className="campaigns__column">{'Installing %'}</div>
            <div className="campaigns__column">{'Not applicable %'}</div>
          </>
        )}
```

Reply:
Done